### ISSUE TITLE
feature request: spawn-proof overlay tweak

### ISSUE THREAD

**Issue by Biplacido** [maruohon/tweakeroo#63]
It would be nice to have a tweak that displays an overlay or indicator if that block is spawn-proof. For example Red if it isn't spawn proof; yellow if hostile mobs could spawn on the block; green if only passive mobs could spawn on it; white if its completely spawn-proof

**Comment by vacla** [comment]
this feature is already in minihud

**Comment by Biplacido** [comment]
what is that?

**Comment by Biplacido** [comment]
is that another client-side mod?

**Comment by vacla** [comment]
yeah, that is another mod from masa, some info about it [](http://curseforge.com/minecraft/mc-mods/minihud)

**Comment by Biplacido** [comment]
ok
